quarkchain/cluster/master.py

```diff
@@ -782,6 +797,15 @@ def __init__(self, env, root_state, name="master"):
 
         self.__init_root_miner()
 
+    def init_grpc_server(self):
+        grpc_server = grpc.server(futures.ThreadPoolExecutor(max_workers=None))
+        servicer = ClusterMaster(self)
+        grpc_pb2_grpc.add_ClusterMasterServicer_to_server(servicer, grpc_server)
+        for s in self.cluster_config.GRPC_SLAVE_LIST:
+            grpc_server.add_insecure_port("{}:{}".format(s.HOST, str(s.PORT)))
```

Comment:
lol if we have 6 grpc slaves, which host and port will be used?

Reply:
all of them will be added into this server as channels. host and port are defined in config file.

Reply:
but as a server you only need to have one host/port to work, right?

Reply:
got it